word_counter.rb

```diff
@@ -0,0 +1,79 @@
+require 'pragmatic_segmenter'
+# how do you include a gem such as this so that the program will run on a given computer?
+
+def word_counter
+	# Lists words, their frequency, and location based on sentence # from given text doc.
+	# The way the file is read, it could freeze the host if file given is too big.
+	# Will not work with languages other than English
+	#
+	# Started 9/16/2015
```

Comment:
All information like this should go in a gemspec, not as a comment in a lib file.

For example, if this project is called `word_counter` (which I think is the name of probably this module, not the whole project), then you'd have a file in your root directory called `word_counter.gemspec` and the contents of that file would look something like:
```
require File.expand_path('../lib/word_counter/version', __FILE__)

Gem::Specification.new do |s|
  s.name = "word_counter"
  s.version = WordCounter::VERSION
  s.summary = "Word Counter does this."
  s.description = "This is a description of Word Counter"
  s.authors = ["Chris Manna"]
  s.email = "Your email"
  s.homepage = "https://github.com/..."
  s.require_paths = ["lib"]

  # If you have any dependencies, add them like this:
  s.add_dependency('activerecord')
  s.add_dependency('activemodel')
  s.add_dependency('activesupport')


  # If you have any development dependencies, then add them like this:
  s.add_development_dependency('rails', '~> 4.2')
  s.add_development_dependency('mysql', '~> 2.9.1')
  s.add_development_dependency('rspec', '~> 3.0')
  s.add_development_dependency('database_cleaner', '1.0.1')

  if 1.9 >  RUBY_VERSION[0..2].to_f
    raise 'Word Counter is only compatible with Ruby >= 1.9.x.'
  elsif '1.9' == RUBY_VERSION[0..2]
    s.add_development_dependency('debugger', '1.6.0')
  else
    s.add_development_dependency('byebug', '~> 4.0')
  end
end
```